src/components/MemoryCard.vue

```diff
@@ -24,15 +24,17 @@
           v-if="isImage"
           draggable="false"
         />
+        <div id="markdown" v-if="isMarkdown" v-html="markdownContent"></div>
```

Comment:
```suggestion
        <div id="markdown" v-else-if="isMarkdown" v-html="markdownContent"></div>
```
Would prefer v-else-if instead of multiple if's

Reply:
I will change this to v-else-if